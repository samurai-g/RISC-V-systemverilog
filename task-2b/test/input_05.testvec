A
11c79c66
de1f5f6c
f534ce98
0355c8df
83760ba5
2cf004ef
d9ddecb4
15a67db9
626728fb
7db64d7b
