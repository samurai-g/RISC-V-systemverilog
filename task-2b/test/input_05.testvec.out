83760ba5
d9ddecb4
de1f5f6c
f534ce98
0355c8df
11c79c66
15a67db9
2cf004ef
626728fb
7db64d7b
